plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/MethodSignatureEmbedding.kt

```diff
@@ -9,27 +9,18 @@ import org.jetbrains.kotlin.formver.conversion.ReturnVariableName
 import org.jetbrains.kotlin.formver.viper.MangledName
 import org.jetbrains.kotlin.formver.viper.ast.*
 
-class MethodSignatureEmbedding(val name: MangledName, val params: List<VariableEmbedding>, val returnType: TypeEmbedding) {
+/**
+ * Some invariants about local variables and parameters are automatically added.
+ * With `extraPreConditions` and `extraPostConditions` you can add additional ones.
+ */
+class MethodSignatureEmbedding(
+    val name: MangledName,
+    val params: List<VariableEmbedding>,
+    val returnType: TypeEmbedding,
+    val pos: Position = Position.NoPosition,
+    val info: Info = Info.NoInfo,
+    val trafos: Trafos = Trafos.NoTrafos,
```

Comment:
I don't think having these makes sense, we never use them anyway... let me make a commit that just removes all this info elsewhere, then we can figure out how to generate it correctly later.  (It doesn't block this commit, though.)